source/client-side-encryption/client-side-encryption.rst

```diff
@@ -838,8 +844,27 @@ Identifies a key vault collection document by 'keyAltName'.
 
 algorithm
 ^^^^^^^^^
-The string "AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic" or
-"AEAD_AES_256_CBC_HMAC_SHA_512-Random"
+One of the strings:
+- "AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic"
+- "AEAD_AES_256_CBC_HMAC_SHA_512-Random"
+- "IndexedEquality"
+- "Unindexed"
+
+The result of explicit encryption with the "IndexedEquality" algorithm must be processed by the server to insert or query. Drivers MUST document the following behavior:
+
+   To insert or query with an "IndexedEquality" encrypted payload, use a ``MongoClient`` configured with ``AutoEncryptionOpts``.
+   ``AutoEncryptionOpts.bypassQueryAnalysis`` may be true. ``AutoEncryptionOpts.bypassAutoEncryption`` must be false.
+
+contentionFactor
+^^^^^^^^^^^^^^^^
+contentionFactor only applies when algorithm is "IndexedEquality".
+It is an error to set contentionFactor when algorithm is not "IndexedEquality".
+Setting a higher contentionFactor adds randomization. It is useful when values being encrypted have low cardinality.
+
+queryType
+^^^^^^^^^
+queryType only applies when algorithm is "IndexedEquality".
+It is an error to set queryType when algorithm is not "IndexedEquality".
```

Comment:
What does it mean when algorithm is IndexedEquality and queryType is unset?

Reply:
> What does it mean when algorithm is IndexedEquality and queryType is unset?

When the algorithm is "IndexedEquality" and the queryType is unset, the result of `ClientEncryption.Encrypt` is suitable for insert.

> What other values are likely to be added in the future (just for my own context, not to add to the spec)

I think DRIVERS-2286 will require a QueryTypeRanged value.